kratos/utilities/variable_utils.h

```diff
@@ -1404,11 +1384,36 @@ class KRATOS_API(KRATOS_CORE) VariableUtils
     ///@name Static Member Variables
     ///@{
 
-
     ///@}
     ///@name Member Variables
     ///@{
 
+    // TODO use SumReduction once it supports array3 (- Philipp)
+    class Array3Reduction
+    {
+    public:
+        typedef array_1d<double,3> value_type;
+        array_1d<double,3> mValue = ZeroVector(3);
```

Comment:
@KratosMultiphysics/technical-committee here is the source of the issues.
Array1D is not default initialized hence we cannot use the `SumReduction` directly

I am working on a global fix but until then we have to do it like this

Reply:
#8289 would fix this

Reply:
Then I'd wait for #8289 to be merged.

Reply:
the point is that I am not sure if #8289 will even be merged, we had a lot of discussions about this in the past
Furthermore I want to make some more unifications / cleanups here and this would unnecessarily block

Reply:
OK. Then makes sense for me that we centralize the array_1d reduction in here.